src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -206,6 +208,9 @@ class CollectedHeap : public CHeapObj<mtGC> {
     return static_cast<T*>(heap);
   }
 
+  // Stop any onging concurrent work and prepare for exit.
+  virtual void stop() = 0;
```

Comment:
Why `= 0` instead of `{}` as before?

Reply:
Both works. I believe it was a stylistic choice.

Reply:
I see; I'd prefer reverting this, as that reduces the diff size, but up to you.